model-server/src/main/kotlin/org/modelix/model/server/handlers/ModelReplicationServer.kt

```diff
@@ -169,8 +169,7 @@ class ModelReplicationServer(
         post<Paths.initializeRepository> {
             fun ApplicationCall.repositoryId() = RepositoryId(parameters["repository"]!!)
             fun PipelineContext<Unit, ApplicationCall>.repositoryId() = call.repositoryId()
-
-            val useRoleIds = call.request.queryParameters["useRoleIds"] != "false"
+            val useRoleIds = call.parseQueryParameterAsBoolean("useRoleIds") ?: return@post
```

Comment:
Alternative could be:

`val useRoleIds = call.request.queryParameters["useRoleIds"]?.toBooleanStrictOrNull() != false`